tests/integration/modules/test_vmc_networks.py

```diff
@@ -6,136 +6,121 @@
 from saltext.vmware.utils import vmc_request
 
 
+@pytest.fixture
+def network_id():
+    return "sample-network"
+
+
 @pytest.fixture
 def subnets():
-    return [{"gateway_address": "30.1.1.1/16"}]
+    return [{"gateway_address": "40.1.1.1/16"}]
```

Comment:
subnets is required only during create call, so i have not included it in common data dict.

network_id is not required for get networks call, so kept it in a separate fixture instead of including it in common data dict.